wad_project/templates/socshare/events.html

```diff
@@ -1,14 +1,21 @@
 {% extends 'socshare/base.html' %}
 {% load static %}
 
+{% block title %}
+Events
+{% endblock %}
+
 {% block container %}
+<h1> Current Events </h1>
```

Comment:
This is redundant af

Reply:
Can be removed, was added in the hopes that I could get past and current events separated.

Reply:
👍